observationtree/src/main/scala/explore/observationtree/ObsBadge.scala

```diff
@@ -43,16 +50,29 @@ object ObsBadge {
     ScalaComponent
       .builder[Props]
       .render_P { props =>
-        val obs = props.obs
+        val obs          = props.obs
+        val deleteButton = Button(
+          size = Small,
+          compact = true,
+          clazz = ExploreStyles.DeleteButton |+| ExploreStyles.ObservationDeleteButton,
+          onClickE = (e: ReactMouseEvent, _: Button.ButtonProps) =>
+            e.preventDefaultCB *> e.stopPropagationCB *> props.deleteCB(props.obs.id)
+        )(
+          Icons.Trash
+        )
 
         <.small(
           Card(raised = props.selected)(ExploreStyles.ObsBadge)(
             CardContent(
               CardHeader(
-                props.layout match {
-                  case NameAndConf        => obs.name.orEmpty
-                  case ConfAndConstraints => obs.conf
-                }
+                <.div(
+                  ExploreStyles.ObservationCardHeader,
+                  props.layout match {
+                    case NameAndConf        => obs.name.orEmpty
+                    case ConfAndConstraints => obs.conf
+                  },
+                  deleteButton
```

Comment:
Is the delete button showing in observations badges in the target view? Do we want that?

Reply:
At the moment yes, it is a good question, I'd say in general yes but it isn't wired at the moment

Reply:
I added a change that will hide the delete button on the targets tree